services/backend/src/core/keycloak/keycloak.js

```diff
@@ -34,12 +34,10 @@ const getGroupIds = async (accessToken) => {
     timeout: 2000,
   });
 
-  const groupIds = groups.data[0].subGroups.map(({ id, name }) => {
-    return {
-      id,
-      name,
-    };
-  });
+  const groupIds = groups.data[0].subGroups.map(({ id, name }) => ({
+    id,
+    name,
+  }));
```

Comment:
is this happening because of the eslint update? I don't mind it, its cleaner but I'm curious

Reply:
Yeah its from the eslint update